templates/resourceclaim.yaml.j2

```diff
@@ -11,7 +11,7 @@ spec:
       kind: AnarchySubject
       metadata:
         annotations:
-          poolboy.gpte.redhat.com/resource-provider-name: babylon
+          poolboy.gpte.redhat.com/resource-provider-name: {{ (engagement_type | lower) + '.' + (governor_type | lower) + '.' + (governor_spec | lower ) }}
```

Comment:
I am not quite sure that this file is in use. Both `inventory-generation/identity-management` (used for day2) and `translate-engagement-data` (ocp4) are having own versions of the `resourceclaim.yaml.j2` template which can be found under `file/templates` directory.

All these `resourceclaim.yaml.j2` files are exact same.

Reply:
I'll raise separate PR for other files